core/capabilities/targets/write_target.go

```diff
@@ -309,7 +310,44 @@ func (cap *WriteTarget) Execute(ctx context.Context, rawRequest capabilities.Cap
 	}
 
 	cap.lggr.Debugw("Transaction submitted", "request", request, "transaction", txID)
-	return capabilities.CapabilityResponse{}, nil
+
+	done := make(chan struct{})
+
+	go func() {
+		defer close(done)
+		for {
+			time.Sleep(100 * time.Millisecond)
```

Comment:
What do you think about an exponential backoff strategy?

Reply:
This should also yield to context while waiting (e.g. `<- time.After(100*time.Millisecond)`) instead of doing a blocking a sleep.